.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+
+jobs:
+  build:
+    macos:
+      xcode: "9.0"
+    working_directory: ~/xcode-install
+    shell: /bin/bash --login -eo pipefail
+    steps:
+      - checkout
+
+      # See Also: https://discuss.circleci.com/t/circleci-2-0-ios-error-installing-gems/23291/4
+      - run:
+          name: Set Ruby Version
+          command: echo "ruby-2.4" > ~/.ruby-version
+
+      - run:
+          name: Install ruby dependencies
+          command: bundle install
+
+      - run:
+          name: Run test
+          command: bundle exec rake spec
+
+      - run:
+          name: Run lint
+          command: bundle exec rake rubocop
+
+workflows:
+  version: 2
+  build_and_test:
+    jobs:
+      - build
```

Comment:
missing new line

Reply:
@KrauseFx Thank you for your review, I've added the missing new line :)